.github/actions/teardown-rocm/action.yml

```diff
@@ -14,13 +14,20 @@ runs:
         docker stop $(docker ps -q) || true
         # Prune all stopped containers.
         docker container prune -f
-        # Prune everything docker if there are more than 10 images (~200GB).
-        # This is easier than using a time filter, e.g., "until=24h".
-        # Might fail if a prune is already in progress by another runner.
-        image_count=$(docker images | wc -l)
-        if [[ ${image_count} -gt 10 ]]; then
-            echo "Purging all docker caches"
-            docker system prune -af || true
```

Comment:
@jeffdaily What was the `|| true` for?

Reply:
To ignore errors I guess

Reply:
It was there to catch errors if another prune was already in progress.  Only needed when we were going to having multiple runners per host.  As it was, having two runners trying to prune each other was a race condition.  I'm glad we went back to just a single runner.  No docker command conflicts.